dns/exported/exported.go

```diff
@@ -0,0 +1,22 @@
+package exported
+
+// Entry is the entry used in the DNS.
+type Entry interface {
+	GetOwners() [][]byte
+	GetContent() []byte
+}
+
+// MsgPreCommitEntry defines the msg interface used to send a pre-commit of a
+// registration in the DNS.
+type MsgPreCommitEntry interface {
```

Comment:
do we need these interface messages? I think the keeper can just refer to the concrete types. I don't think anyone would need to make their own custom precommitentry message? The entry interface makes sense though